.gitignore

```diff
@@ -57,8 +58,6 @@ zulip-git-version
 
 ## Files (or really symlinks) created in a prod deployment
 /zproject/prod_settings.py
-/zulip-current-venv
-/zulip-py3-venv
```

Comment:
Hmm, do we need to keep these, so if you're switching branches across the transition, these don't show up as new/untracked files? 

I guess `provision` on the old branch will re-create them, so maybe it's fine if `provision` on the new branch deletes...

Reply:
zulip-current-venv was production-only. I’ll have `provision` delete `zulip-py3-venv`.